src/exec/run-batch-submitter.ts

```diff
@@ -37,7 +37,7 @@ interface RequiredEnvVars {
   MAX_BATCH_SIZE: 'MAX_BATCH_SIZE'
   // The maximum amount of time (seconds) that we will wait before submitting an under-sized batch.
   MAX_BATCH_SUBMISSION_TIME: 'MAX_BATCH_SUBMISSION_TIME'
-  // The delay in seconds between querying L2 for more transactions / to create a new batch.
+  // The delay in milliseconds between querying L2 for more transactions / to create a new batch.
   POLL_INTERVAL: 'POLL_INTERVAL'
```

Comment:
Something I've found useful in the past is to add the unit as a suffix to the variable name. e.g.,
- `MAX_BATCH_SUBMISSION_TIME_S`
- `POLL_INTERVAL_MS`